licenses/spdx/BUILD

```diff
@@ -38,9 +52,15 @@
 # If you maintain your own  third_party directory of all your dependences,
 # so that your WORKSPACE files are empty, you might favor the second.
 
-
+load("@rules_license//rules:license.bzl", "license")
```

Comment:
The `license` rule is loaded but is unused, this line can be removed.
```suggestion
```

Reply:
done